LoopKitUI/InsulinKit/InsulinDeliveryTableViewController.swift

```diff
@@ -16,15 +16,30 @@ public final class InsulinDeliveryTableViewController: UITableViewController {
 
     @IBOutlet var needsConfigurationMessageView: ErrorBackgroundView!
 
-    @IBOutlet weak var iobValueLabel: UILabel!
+    @IBOutlet weak var iobValueLabel: UILabel! {
+        didSet {
+            iobValueLabel.textColor = headerValueLabelColor
```

Comment:
[nit] Why not just use `.label` directly as you do for `cell.textLabel?.textColor` on line 384 below?

Reply:
This colour is now configurable by Loop